test/unit_tests/test_interface.py

```diff
@@ -1,11 +1,35 @@
 from unittest.case import TestCase
+from unittest.mock import patch
 from unittest.mock import Mock
 
 from com_redhat_kdump import common
+from .mock import MockBuiltinRead
 from com_redhat_kdump.constants import KDUMP
 from com_redhat_kdump.service.kdump import KdumpService
 from com_redhat_kdump.service.kdump_interface import KdumpInterface
 
+SYS_CRASH_SIZE = '/sys/kernel/kexec_crash_size'
+PROC_MEMINFO = '/proc/meminfo'
+CRASHKERNEL_DEFAULT = '/usr/lib/modules/{}/crashkernel.default'
```

Comment:
Btw, the code related to CRASHKERNEL_DEFAULT has removed. So it's used nowhere. You should delete this it.